src/libraries/System.Net.Primitives/ref/System.Net.Primitives.cs

```diff
@@ -323,6 +323,24 @@ public abstract partial class TransportContext
         protected TransportContext() { }
         public abstract System.Security.Authentication.ExtendedProtection.ChannelBinding? GetChannelBinding(System.Security.Authentication.ExtendedProtection.ChannelBindingKind kind);
     }
+    public enum NetworkError : int
+    {
+        Unknown = 0,
+        AddressInUse,
+        InvalidAddress,
```

Comment:
"address" makes sense in terms of `struct sockaddr`, but we should probably use `EndPoint` here as the API has nothing called address. Thoughts?

```suggestion
        EndPointInUse,
        InvalidEndPoint,
```

Reply:
Sounds good, but this will require changing the places it's used as well :)